cypress/e2e/calendar/edit-existing-calendars.cy.js

```diff
@@ -0,0 +1,100 @@
+import {
+  MultiColumnListCell, MultiColumnListRow, Pane, Button, Accordion, TextField,
+  including, Link
+} from '../../../interactors';
+import calendarFixtures from '../../fixtures/calendar_e2e_fixtures';
+
+const testServicePoint = calendarFixtures.servicePoint;
+const testCalendar = calendarFixtures.calendar;
+
+// delete all but the first opening time
+testCalendar.normalHours.splice(1);
+
+const editExistingCalendarsData = calendarFixtures.data.editExistingCalendars;
+
+
+describe('Edit existing calendars', () => {
+  let testCalendarResponse;
+  before(() => {
+    // login
+    cy.loginAsAdmin();
+
+    // reset db state
+    cy.deleteServicePoint(testServicePoint.id, false);
+
+    // create test service point
+    cy.createServicePoint(testServicePoint, (response) => {
+      testCalendar.assignments = [response.body.id];
+
+      cy.createCalendar(testCalendar, (calResponse) => {
+        testCalendarResponse = calResponse.body;
+      });
+      cy.openCalendarSettings();
+    });
+
+    cy.openCalendarSettings();
+  });
+
+  after(() => {
+    cy.deleteServicePoint(testServicePoint.id, true);
+    cy.deleteCalendar(testCalendarResponse.id);
+  });
+
+
+  it('edits a single existing calendar', () => {
```

Comment:
it('{test case id from test rail} {test case name from test rail} (vega)', { tags: [TestTypes.smoke, devTeams.vega] }, () => {